src/main/resources/templates/db_scripts/schema.json

```diff
@@ -105,6 +105,11 @@
       "run": "after",
       "snippetPath": "transaction_totals_view.sql",
       "fromModuleVersion": "mod-finance-storage-8.8.0"
+    },
+    {
+      "run": "after",
+      "snippetPath": "job_number_sequence.sql",
```

Comment:
we had issues previously with sequences and become to use table approach, you can take a look into orders_storage po_number table. I will post ticket why we become to use table approach.
Also we can intorduce even one more column to this table like job Id and number. In this case our new API will be universal

Reply:
https://folio-org.atlassian.net/browse/MODORDSTOR-394 

https://www.cybertec-postgresql.com/en/gaps-in-sequences-postgresql/

Reply:
Used table approach, thanks